### PR TITLE
Psp 5584 form tab on acq file, ability to add a form based on type to acq file

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: source/backend/api/Models/Concepts/ActivityInstance/ActivityInstanceFileModel.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 namespace Pims.Api.Models.Concepts
 {
-    public class ActivityInstanceFileModel
+    public class ActivityInstanceFileModel : BaseAppModel
```
Comment (FuriousLlama) [inline source/backend/api/Models/Concepts/ActivityInstance/ActivityInstanceFileModel.cs:3]: is this change related to activities?

File: source/backend/api/Models/Concepts/Form/FileFormModel.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace Pims.Api.Models.Concepts
+{
+    public class FileFormModel : BaseAppModel
+    {
+        public long Id { get; set; }
+
+        public long FileId { get; set; }
+
+        public LookupModel<string> FormTypeCode { get; set; }
```
Comment (FuriousLlama) [inline source/backend/api/Models/Concepts/Form/FileFormModel.cs:11]: Not sure the form type matches the lookup exactly given it also has a document id

File: source/frontend/src/features/properties/map/form/hooks/useFormRepository.tsx
Comment (FuriousLlama) [inline source/frontend/src/features/properties/map/form/hooks/useFormRepository.tsx:1]: Lets move repositories that interact with the api to the /hook/repositories folder instead of being part of the feature path

File: source/backend/api/Models/Concepts/Form/FileFormModel.cs (hunk 2 above)
Comment (devinleighsmith) [reply source/backend/api/Models/Concepts/Form/FileFormModel.cs:11]: Oh, right. Its a lookup with that one extra attribute. All right if I fix that in my next PR? will probably be easier.

Comment (FuriousLlama) [reply source/backend/api/Models/Concepts/Form/FileFormModel.cs:11]: I will change it once it I merge it.

**Review by FuriousLlama — APPROVED** [review]
